data/competencies.yml

```
@@ -925,3 +925,15 @@
   level: senior1-to-senior2
   area: leadership
   domain: null
+
+
+- id: f173b7e9-1c46-4afd-a659-d264aa173759
+  summary: Contributes to inclusive organisational culture
+  examples:
+    - Membership of internal groups that promote diversity and inclusion (such as FT Embrace, FT Proud, Mental health first aiders)
```

I'm wondering if "Membership of" should be re-worded to "Participation in" because you might not be a committee member but still attend and support the events organised by those groups.

---

I'm happy with this suggestion, thanks Lily. Also including @nickcolley's suggestion to include FT Access.

```suggestion
    - Participates in internal groups that promote diversity and inclusion (such as FT Access, FT Embrace, FT Proud, Mental health first aiders)
```